crates/rome_formatter/tests/prettier_tests.rs

```diff
@@ -0,0 +1,112 @@
+use parking_lot::{const_mutex, Mutex};
+use similar::{utils::diff_lines, Algorithm};
+use std::{
+    env,
+    ffi::OsStr,
+    fmt::Write,
+    fs::{read_to_string, write},
+    path::Path,
+};
+
+use rome_formatter::{FormatOptions, IndentStyle};
+use rslint_parser::parse_script;
+
+static REPORTER: DiffReport = DiffReport::new();
+
+tests_macros::gen_tests! {"tests/specs/prettier/**/*.js", test_snapshot, "script"}
```

Comment:
```suggestion
tests_macros::gen_tests! {"tests/specs/prettier/**/**/**/*.js", test_snapshot, "script"}
```

I don't know how deep is the folder structure of prettier tests, but maybe it's safer to specify a deeper level?

Reply:
The `**` glob expression will recurse into directories while performing the search so there's no need to specify it multiple times

Reply:
That's the thing. I doesn't work as expected then. Sometimes I create a new level of directories and it's not recognized unless I add a new `**/`

Reply:
I think that's not an issue with the glob search but with how Cargo is caching builds: because it doesn't know the tests depend on these external files it will not re-build the tests (and re-run the search in the proc macro) until the `.rs` file itself is modified